README.md

```diff
@@ -1,5 +1,5 @@
 # CSI WekaFS Driver
-![Version: 0.7.3](https://img.shields.io/badge/Version-0.7.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.7.3](https://img.shields.io/badge/AppVersion-v0.7.3-informational?style=flat-square)
+![Version: 0.7.4-beta](https://img.shields.io/badge/Version-0.7.4--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.7.4-beta](https://img.shields.io/badge/AppVersion-v0.7.4--beta-informational?style=flat-square)
```

Comment:
why beta in code? 
beta is manually created release...code should contain only actual versions

Reply:
Will drop this commit